case-studies/blablacar.html

```diff
@@ -0,0 +1,99 @@
+---
+title: BlaBlaCar Case Study
+layout: basic
+case_study_styles: true
+cid: caseStudies
+css: /css/style_blablacar.css
+---
+
+<div class="banner1">
+  <h1> CASE STUDY:<img src="/images/blablacar_logo.png" class="header_logo"><br> <div class="subhead">Turning to Containerization to Support Millions of Rideshares</div></h1>
+
+</div>
+
+<div class="details">
+    Company &nbsp;<b>BlaBlaCar</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Location &nbsp;<b>Paris, France</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Industry &nbsp;<b>Ridesharing Company</b>
+</div>
+
+<hr>
+
+<section class="section1">
+<div class="cols">
+  <div class="col1">
+    <h2>Challenge</h2>
+        The world’s largest long-distance carpooling community, <a href="https://www.blablacar.com/">BlaBlaCar</a>, connects 40 million members across 22 countries. The company has been experiencing exponential growth since 2012 and needed its infrastructure to keep up. "When you’re thinking about doubling the number of servers, you start thinking, ‘What should I do to be more efficient?’" says Simon Lallemand, Infrastructure Engineer at BlaBlaCar. "The answer is not to hire more and more people just to deal with the servers and installation." The team knew they had to scale the platform, but wanted to stay on their own bare metal servers.
+    <br>
+      <h2>Solution</h2>
```

Comment:
The layout in the output isn't quite right here. The HTML is at least one culprit -- it looks as though you need one more `<br>` tag, per comparison with other case studies. So you'd have:
`<h2>Challenge</h2>
    The world's largest ...
<br><br>
<h2>Solution</h2>
    ...`

Reply:
Ah - thanks for catching that! I totally glanced over it. Updated the HTML.